src/OpenTelemetry.Exporter.Geneva/Internal/Transports/UnixUserEventsDataTransport.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#if NET
+
+using Microsoft.LinuxTracepoints.Provider;
+
+namespace OpenTelemetry.Exporter.Geneva.Transports;
+
+internal sealed class UnixUserEventsDataTransport : IDisposable
```

Comment:
Is there a plan to implement `IDataTransport` later?

Reply:
This is interesting. For metrics we send more or less the same otlp byte[] blob: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/3124e538821054b356ec5b739e9225d948b73e08/src/OpenTelemetry.Exporter.Geneva/Metrics/Transport/MetricUnixUserEventsDataTransport.cs#L54

Logs needs something different?

Reply:
Yes. Logs do not use OTLP byte array but use the EventHeader which is a component of the user_events library.